recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "orange-widget-base" %}
-{% set version = "4.17.0" %}
+{% set version = "4.19.0" %}
```

Comment:
Is version 4.20.0 suitable for us too?

Reply:
It probably is, however I was thinking of not upgrading to the latest and greatest since it may introduce new dependencies and further complicate the build. For the moment I'm trying to get this out the door as soon as possible. Feel free to disagree.

Reply:
upgrading to 4.20.0 -> orange-canvas-core 0.1.30 -> updating anyqt from 0.1.0 -> 0.2.0 which is very burdensome.